### DISCUSSION TITLE
Dark mode?

### DISCUSSION THREAD

**Discussion by nxvvvv** [upptime/upptime#219] [Q&A] [answered]
- How can I get dark mode in my uptime page
- How to edit the page, I tried editing gh-pages branch but nothing happened. I need assistance in that

**Comment by AnandChowdhary** [comment]
I'll keep you posted, we're adding first-class dark mode support soon!

**Reply by nxvvvv** [reply]
okie

**Comment by nxvvvv** [comment]
@AnandChowdhary How to edit the page, I tried editing gh-pages branch but nothing happened. I need assistance in that

**Reply by AnandChowdhary** [reply]
What do you mean by editing the page? You can fork the status website repository to generate your site from scratch, otherwise you're limited to the configuration options in `.upptimerc.yml`.

**Reply by nxvvvv** [reply]
ohh

**Comment by nxvvvv** [comment]
I meant editing the webpage

**Comment by nxvvvv** [comment]
If i edit gh-pages branch, will it change?

**Comment by nxvvvv** [comment]
Or how can i change the home page?

**Comment by nxvvvv** [comment]
Bro I made a dark one by editing css. But I'm not able to apply. 
![image](https://user-images.githubusercontent.com/34748927/104992334-cf79ab00-5a46-11eb-9733-3a7a4cd4aab1.png)

**Comment by nxvvvv** [comment]
![image](https://user-images.githubusercontent.com/34748927/104992585-647ca400-5a47-11eb-9766-111b818686b0.png)

**Comment by nxvvvv** [comment]
@AnandChowdhary How can i edit it like this?

**Comment by nxvvvv** [comment]
So trial and error method is the way. I understand that I won't get help in this part.

**Reply by AnandChowdhary** [reply]
Repeating my previous answer here:

> What do you mean by editing the page? You can fork the status website repository to generate your site from scratch, otherwise you're limited to the configuration options in .upptimerc.yml.

**Reply by nxvvvv** [reply]
Ok! Thanks!

**Comment by AnandChowdhary** [comment, marked as answer]
Support for themes has been added in [`upptime/status-page@v1.8.0`](https://github.com/upptime/status-page/releases/tag/v1.8.0). Theme can be `light`, `dark`, or `ocean`:

```yaml
status-page:
  theme: "light"
```

| Dark | Ocean |
| - | - |
| ![Screen Shot 2021-01-22 at 20 14 22](https://user-images.githubusercontent.com/2841780/105506420-09b9a580-5cf0-11eb-81d3-2c84f3b945d8.png) | ![Screen Shot 2021-01-22 at 20 16 25](https://user-images.githubusercontent.com/2841780/105506404-045c5b00-5cf0-11eb-89f8-c50ab2f7f98e.png) |

You can create your own themes using CSS variables, using [`dark.css`](https://github.com/upptime/status-page/blob/v1.8.0/static/themes/dark.css), for example as a starting point:

```css
:root {
  --body-background-color: #001716;
  --body-text-color: #f0ffff;
  --card-background-color: #002b29;
  --nav-background-color: #002b29;
  --nav-border-bottom-color: #015450;
}
/* . . . */
```

**Comment by nxvvvv** [comment]
> Support for themes has been added in [`upptime/status-page@v1.8.0`](https://github.com/upptime/status-page/releases/tag/v1.8.0). Theme can be `light`, `dark`, or `ocean`:
> 
> ```yaml
> status-page:
>   theme: "light"
> ```
> 
> | Dark | Ocean |
> | - | - |
> | ![Screen Shot 2021-01-22 at 20 14 22](https://user-images.githubusercontent.com/2841780/105506420-09b9a580-5cf0-11eb-81d3-2c84f3b945d8.png) | ![Screen Shot 2021-01-22 at 20 16 25](https://user-images.githubusercontent.com/2841780/105506404-045c5b00-5cf0-11eb-89f8-c50ab2f7f98e.png) |
> 
> You can create your own themes using CSS variables, using [`dark.css`](https://github.com/upptime/status-page/blob/v1.8.0/static/themes/dark.css), for example as a starting point:
> 
> ```css
> :root {
>   --body-background-color: #001716;
>   --body-text-color: #f0ffff;
>   --card-background-color: #002b29;
>   --nav-background-color: #002b29;
>   --nav-border-bottom-color: #015450;
> }
> /* . . . */
> ```

Niceee.
